lib/misc/blocks.coffee

```diff
@@ -0,0 +1,56 @@
+# TODO: docstrings
+
+scopes = require './scopes'
+
+module.exports =
+
+  getLine: (ed, l) ->
+    ed.getTextInBufferRange [[l, 0], [l, Infinity]]
```

Comment:
If this works reliably, we should probably replace all occurences of creating ranges that span a complete line with it instead of explicitly getting the line length.